README.md

```diff
@@ -192,7 +196,23 @@ already implemented one that matches your need.
    your TokenizerWrapper subclass. Otherwise, you can directly use TokenizerWrapper.
 
 
-5) **transformers.Pipeline**:
+5) **MetricHandler**:
+    This class is responsible for postprocess operations applied to prediction outputs.
+    In this particular class it is needed to transform the prediction outputs to a
+    suitable format to be fed in metrics for computation. For example, using BLEU in a 
+    language generation task, the prediction outputs need to be converted to string or
+    list of strings, which is the behavior of the default implementation. However, for
```

Comment:
I don't think it should be the default implementation. The default should be pass-through.